File: 2/count_vowels.py
import sys


def main():
    vowels = 'aeiouAEIOU'
    vowel_count = 0
    s = sys.argv[1]
    for c in s:
        if c in vowels:
            vowel_count += 1
    print vowel_count

if __name__ == '__main__':
    main()

File: 2/CountVowels.java
public class CountVowels {
    
    public static boolean isVowel(char c){
        String vowels = "aeiouAEIOU";
        return vowels.indexOf(c) != -1;
}
    public static void main(String[] args) {
        
        String s = args[0];
        int vowelCount = 0;
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if (isVowel(c)){
                vowelCount++;
            }
        }
        System.out.println(" number of vowels :" + vowelCount );
    }
    
}